packages/languages/src/browser/language-client-contribution.ts

```diff
@@ -59,11 +61,26 @@ export abstract class BaseLanguageClientContribution implements LanguageClientCo
     }
 
     waitForActivation(app: FrontendApplication): Promise<any> {
+        const activationPromises: Promise<any>[] = [];
+        const workspaceContains = this.workspaceContains;
+        if (workspaceContains) {
```

Comment:
the guard is redundant, `workspaceContains` never can be undefined, maybe you meant `workspaceContains.length !== 0`?